_utils/routes-listing.ts

```diff
@@ -2,10 +2,10 @@ import {
 	adoptedSchema,
 	adoptSchema,
 	AjvSchema,
-	allSchema,
-	byageSchema,
+	// allSchema,
```

Comment:
Same here. These routes will probably never come back. Maybe we should delete the lines, instead of commenting them out.

Reply:
Yes. You are right. Removed it